pkg/validators/validator_addon_label.go

```diff
@@ -5,11 +5,12 @@ import (
 )
 
 // ValidateAddonLabel validates whether the 'label' field under an addon.yaml follows the format 'api.openshift.com/addon-<id>'
-func ValidateAddonLabel(metabundle utils.MetaBundle) (bool, error) {
+// TODO - remove this validator once we do field level validation
+func ValidateAddonLabel(metabundle utils.MetaBundle) (bool, string, error) {
 	operatorId, label := metabundle.AddonMeta.ID, metabundle.AddonMeta.Label
 	if label != "api.openshift.com/addon-"+operatorId {
-		return false, nil
+		return false, "", nil
```

Comment:
```suggestion
		return false, "fmt.Sprintf("addon label '%s' wasn't recognised to follow the 'api.openshift.com/addon-<id>' format", label)", nil
```
how does this sound? :)

Reply:
fixed